torchdata/datapipes/iter/load/online.py

```diff
@@ -65,18 +65,23 @@ class HTTPReaderIterDataPipe(IterDataPipe[Tuple[str, StreamWrapper]]):
         **kwargs: a Dictionary to pass optional arguments that requests takes. For the full list check out https://docs.python-requests.org/en/master/api/
 
     Example:
-        >>> from torchdata.datapipes.iter import IterableWrapper, HttpReader
-        >>> file_url = "https://raw.githubusercontent.com/pytorch/data/main/LICENSE"
-        >>> query_params = {"auth" : ("fake_username", "fake_password"), "allow_redirects" : True}
-        >>> timeout = 120
-        >>> http_reader_dp = HttpReader(IterableWrapper([file_url]), timeout=timeout, query_params)
-        >>> reader_dp = http_reader_dp.readlines()
-        >>> it = iter(reader_dp)
-        >>> path, line = next(it)
-        >>> path
-        https://raw.githubusercontent.com/pytorch/data/main/LICENSE
-        >>> line
-        b'BSD 3-Clause License'
+
+    .. testcode::
+
+        from torchdata.datapipes.iter import IterableWrapper, HttpReader
+
+        file_url = "https://raw.githubusercontent.com/pytorch/data/main/LICENSE"
+        query_params = {"auth" : ("fake_username", "fake_password"), "allow_redirects" : True}
+        timeout = 120
+        http_reader_dp = HttpReader(IterableWrapper([file_url]), timeout=timeout, **query_params)
+        reader_dp = http_reader_dp.readlines()
+        it = iter(reader_dp)
+        path, line = next(it)
+        print((path, line))
+
+    .. testoutput::
+
+        ('https://raw.githubusercontent.com/pytorch/data/main/LICENSE', b'BSD 3-Clause License')
```

Comment:
It might be a good idea to add a headline like "Output:" (analogous to "Example:" above). Appears more than one time in this PR.

Reply:
Are those results shown in the output?

Reply:
I mean is there way that we can preserve the format of
```py
>>> code
result
```
and test the output?

Reply:
Yes, the output of the print statement is shown in the example.

![grafik](https://user-images.githubusercontent.com/122370631/220964745-b0438f74-f4ff-4143-ae10-7f5c4a23e8bd.png)

Reply:
Unfortunately we cannot preserve code, output, code, output as the output blocks are rendered separately as seen in the screenshot and have quite some spacing around. This is why I switched to using tuples.